examples/app_commands/basic.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+class MyBot(commands.Bot):
```

Comment:
For this example, the bare bones basic, I think it makes sense to just use a regular `discord.Client` instance instead of `commands.Bot` since we don't want to complicate it. Other examples can use `commands.Bot` such as `cog.py`.